### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by darxriggs**

### CODE REVIEW

**Comment by darxriggs** [comment]
@oleg-nenashev Can you also merge this pull request?

I am already preparing more pull requests.

**Comment by oleg-nenashev** [comment]
I am not a maintainer here AFAIK :(

**Comment by darxriggs** [comment]
@ikedam are you still maintaining this plugin or who can merge pull requests?

File: src/test/java/org/jenkinsci/plugins/authorizeproject/strategy/SpecificUsersAuthorizationStrategyTest.java [hunk 1]
```diff
@@ -329,7 +329,7 @@ public void testLoadOnStart() throws Exception {
     }
     
     private String getConfigXml(XmlPage page) throws TransformerException {
-        // {@link XmlPage#asXml} does unneccessary indentations.
+        // {@link XmlPage#asXml} does unnecessary indentations.
```
Comment (ikedam) [inline src/test/java/org/jenkinsci/plugins/authorizeproject/strategy/SpecificUsersAuthorizationStrategyTest.java]: “unnecessary” often confuses me...

**Pull request merged**